posthog/migrations/0198_property_definition_timestamps.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.2.5 on 2022-01-19 23:51
+
+import json
+
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+from django.db.models import Case, Q, When
+
+from posthog.constants import AnalyticsDBMS
+
+
+def set_created_at(apps, schema_editor):
+    try:
+        from ee.clickhouse.client import sync_execute
+    except ImportError:
+        sync_execute = None  # type: ignore
+
+    EventDefinition = apps.get_model("posthog", "EventDefinition")
+    PropertyDefinition = apps.get_model("posthog", "PropertyDefinition")
+
+    for instance in EventDefinition.objects.all():
+        created_at = None
+        property_keys = []
+        if settings.PRIMARY_DB == AnalyticsDBMS.POSTGRES:
```

Comment:
Curious why we're still supporting Postgres here, I think we should assume the user is on Clickhouse.